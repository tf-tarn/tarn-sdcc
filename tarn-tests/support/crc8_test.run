./sim/sim -q -h -s 90000 $program_file < $input > $output
