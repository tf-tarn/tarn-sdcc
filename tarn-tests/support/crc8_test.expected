OUTPUT=00
OUTPUT=07
OUTPUT=0e
OUTPUT=09
OUTPUT=1c
OUTPUT=1b
OUTPUT=12
OUTPUT=15
OUTPUT=38
OUTPUT=3f
OUTPUT=36
OUTPUT=31
OUTPUT=24
OUTPUT=23
OUTPUT=2a
OUTPUT=2d
OUTPUT=70
OUTPUT=77
OUTPUT=7e
OUTPUT=79
OUTPUT=6c
OUTPUT=6b
OUTPUT=62
OUTPUT=65
OUTPUT=48
OUTPUT=4f
OUTPUT=46
OUTPUT=41
OUTPUT=54
OUTPUT=53
OUTPUT=5a
OUTPUT=5d
OUTPUT=e0
OUTPUT=e7
OUTPUT=ee
OUTPUT=e9
OUTPUT=fc
OUTPUT=fb
OUTPUT=f2
OUTPUT=f5
OUTPUT=d8
OUTPUT=df
OUTPUT=d6
OUTPUT=d1
OUTPUT=c4
OUTPUT=c3
OUTPUT=ca
OUTPUT=cd
OUTPUT=90
OUTPUT=97
OUTPUT=9e
OUTPUT=99
OUTPUT=8c
OUTPUT=8b
OUTPUT=82
OUTPUT=85
OUTPUT=a8
OUTPUT=af
OUTPUT=a6
OUTPUT=a1
OUTPUT=b4
OUTPUT=b3
OUTPUT=ba
OUTPUT=bd
OUTPUT=c7
OUTPUT=c0
OUTPUT=c9
OUTPUT=ce
OUTPUT=db
OUTPUT=dc
OUTPUT=d5
OUTPUT=d2
OUTPUT=ff
OUTPUT=f8
OUTPUT=f1
OUTPUT=f6
OUTPUT=e3
OUTPUT=e4
OUTPUT=ed
OUTPUT=ea
OUTPUT=b7
OUTPUT=b0
OUTPUT=b9
OUTPUT=be
OUTPUT=ab
OUTPUT=ac
OUTPUT=a5
OUTPUT=a2
OUTPUT=8f
OUTPUT=88
OUTPUT=81
OUTPUT=86
OUTPUT=93
OUTPUT=94
OUTPUT=9d
OUTPUT=9a
OUTPUT=27
OUTPUT=20
OUTPUT=29
OUTPUT=2e
OUTPUT=3b
OUTPUT=3c
OUTPUT=35
OUTPUT=32
OUTPUT=1f
OUTPUT=18
OUTPUT=11
OUTPUT=16
OUTPUT=03
OUTPUT=04
OUTPUT=0d
OUTPUT=0a
OUTPUT=57
OUTPUT=50
OUTPUT=59
OUTPUT=5e
OUTPUT=4b
OUTPUT=4c
OUTPUT=45
OUTPUT=42
OUTPUT=6f
OUTPUT=68
OUTPUT=61
OUTPUT=66
OUTPUT=73
OUTPUT=74
OUTPUT=7d
OUTPUT=7a
OUTPUT=89
OUTPUT=8e
OUTPUT=87
OUTPUT=80
OUTPUT=95
OUTPUT=92
OUTPUT=9b
OUTPUT=9c
OUTPUT=b1
OUTPUT=b6
OUTPUT=bf
OUTPUT=b8
OUTPUT=ad
OUTPUT=aa
OUTPUT=a3
OUTPUT=a4
OUTPUT=f9
OUTPUT=fe
OUTPUT=f7
OUTPUT=f0
OUTPUT=e5
OUTPUT=e2
OUTPUT=eb
OUTPUT=ec
OUTPUT=c1
OUTPUT=c6
OUTPUT=cf
OUTPUT=c8
OUTPUT=dd
OUTPUT=da
OUTPUT=d3
OUTPUT=d4
OUTPUT=69
OUTPUT=6e
OUTPUT=67
OUTPUT=60
OUTPUT=75
OUTPUT=72
OUTPUT=7b
OUTPUT=7c
OUTPUT=51
OUTPUT=56
OUTPUT=5f
OUTPUT=58
OUTPUT=4d
OUTPUT=4a
OUTPUT=43
OUTPUT=44
OUTPUT=19
OUTPUT=1e
OUTPUT=17
OUTPUT=10
OUTPUT=05
OUTPUT=02
OUTPUT=0b
OUTPUT=0c
OUTPUT=21
OUTPUT=26
OUTPUT=2f
OUTPUT=28
OUTPUT=3d
OUTPUT=3a
OUTPUT=33
OUTPUT=34
OUTPUT=4e
OUTPUT=49
OUTPUT=40
OUTPUT=47
OUTPUT=52
OUTPUT=55
OUTPUT=5c
OUTPUT=5b
OUTPUT=76
OUTPUT=71
OUTPUT=78
OUTPUT=7f
OUTPUT=6a
OUTPUT=6d
OUTPUT=64
OUTPUT=63
OUTPUT=3e
OUTPUT=39
OUTPUT=30
OUTPUT=37
OUTPUT=22
OUTPUT=25
OUTPUT=2c
OUTPUT=2b
OUTPUT=06
OUTPUT=01
OUTPUT=08
OUTPUT=0f
OUTPUT=1a
OUTPUT=1d
OUTPUT=14
OUTPUT=13
OUTPUT=ae
OUTPUT=a9
OUTPUT=a0
OUTPUT=a7
OUTPUT=b2
OUTPUT=b5
OUTPUT=bc
OUTPUT=bb
OUTPUT=96
OUTPUT=91
OUTPUT=98
OUTPUT=9f
OUTPUT=8a
OUTPUT=8d
OUTPUT=84
OUTPUT=83
OUTPUT=de
OUTPUT=d9
OUTPUT=d0
OUTPUT=d7
OUTPUT=c2
OUTPUT=c5
OUTPUT=cc
OUTPUT=cb
OUTPUT=e6
OUTPUT=e1
OUTPUT=e8
OUTPUT=ef
OUTPUT=fa
OUTPUT=fd
OUTPUT=f4
OUTPUT=f3
OUTPUT=f3
OUTPUT=f3
OUTPUT=f3
OUTPUT=f3
OUTPUT=f3
OUTPUT=f3
OUTPUT=f3
OUTPUT=f3
OUTPUT=f3
OUTPUT=f3
OUTPUT=f3
OUTPUT=f3
OUTPUT=f3
OUTPUT=f3
OUTPUT=f3
OUTPUT=f3
OUTPUT=f3
OUTPUT=f3
OUTPUT=f3
OUTPUT=f3
OUTPUT=f3
OUTPUT=f3
